index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<head>
+  <title>Lake Placid</title>
+  <meta name="viewport" content="width=device-width" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+    rel="stylesheet"
+  />
+  <link rel="stylesheet" href="stylesheet.css" />
+  <script src="scripts.js"></script>
```

Comment:
Great work with indentation in this html file. 
Add "defer" in the script tag for script.js.
A script that will be downloaded in parallel to parsing the page, and executed after the page has finished parsing:

example:
<script src="scripts.js" defer></script>